feat: add update-delete-ignore-not-found example (#40)
prisma/prisma-client-extensions: update-delete-ignore-not-found/README.md

# Prisma Client Extension - Update and Delete ignore on not found

> This example leverages Prisma Client extensions to enhance your models with `updateIgnoreNotFound` and `deleteIgnoreNotFound` functions. These functions ensures that if an `update` or `delete` operation is performed on a field that doesn't exist,  `null` is returned, instead of throwing an error.

Reviewer: ```suggestion
This example adds the `updateIgnoreNotFound` and `deleteIgnoreNotFound` methods to all your models. The two methods allow you to return `null` if a record is not found when updating or deleting a record instead of throwing an error.
```

Please read [the documentation on `model` extensions](https://www.prisma.io/docs/concepts/components/prisma-client/client-extensions/model) for more information.

## How to use

### Prerequisites

- Install [Node.js](https://nodejs.org/en/download/)

### 1. Download example & install dependencies

Clone this repository:

```sh
git clone git@github.com:prisma/prisma-client-extensions.git
```

Install dependencies:

```sh
cd update-delete-ignore-not-found
npm install
```

### 2. Create an SQLite database and run migrations

Run the following command. An SQLite database will be created automatically:

```sh
npx prisma migrate deploy
```

### 3. Seed the database

Run the following command to add seed data to the database:

```sh
npx prisma db seed

> ```

Reviewer: ````suggestion
### 2. Create an SQLite database and run migrations

Run the following command to create a SQLite database and seed your database with sample data:

```sh
npx prisma migrate dev
```
````

> ### 4. Run the `dev` script

Reviewer: ```suggestion
### 3. Run the `dev` script
```

Depends on if the previous suggestion is accepted
